static/variables.less

```diff
@@ -11,7 +11,7 @@
 @twitter-color: #1da1f2;
 
 /*----- LAYOUT -----*/
-@brand-height: 53px;
+@brand-height: 55px;
```

Comment:
As far as I can tell, the header has never _actually_ been 53px. It’s 55px tall.